bin/agitator

```diff
@@ -17,6 +17,7 @@
 
 bin_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 at_home=$( cd "$( dirname "$bin_dir" )" && pwd )
+source "${bin_dir}/build"
```

Comment:
Do we want to source this file, or execute it? (source will insert any variables set in the build command into the current execution environment, vs. execute, which will treat them as local to the build command).

Reply:
build is sourcing $at_home/conf/env.sh so I assume a source is required?

Reply:
Typically, sourcing an env file makes sense because the the whole point of an env script is to create environment variables that the caller wants to include in its own environment.

I don't know enough about the build command to know if it is intended to behave the same way as an env script or if it's supposed to be merely executing and returning. If it's the latter, you don't need to source it.

Reply:
The build script is basically just doing a ```source env.sh``` and then env.sh will build the shaded jar.

Reply:
In that case, it needs to be sourced, but the build name is just confusing, since all it's doing is sourcing the env script.